samples/dubbogo/multi/config/conf.yaml

```diff
@@ -20,16 +20,12 @@
 static_resources:
   listeners:
     - name: "net/http"
+      protocol_type: "HTTP"
       address:
         socket_address:
-          protocol_type: "HTTP"
           address: "0.0.0.0"
           port: 8882
       filter_chains:
-        - filter_chain_match:
-          domains:
-            - api.dubbo.com
-            - api.pixiu.com
           filters:
             - name: dgp.filter.httpconnectionmanager
```

Comment:
as above, using its abbrevlation instead.

Reply:
following envoy http_connection_manager. hcm or gcm for grpcconnectionmanager make too much information lost